src/sentry/api/endpoints/project_repo_path_parsing.py

```diff
@@ -130,6 +133,12 @@ def post(self, request: Request, project) -> Response:
         source_url = data["source_url"]
         stack_path = data["stack_path"]
 
+        # If stack_path uses backslashes, convert them to forward slashes for find_root function
+        is_backslash_stack_path = False
+        if "\\" in stack_path:
+            is_backslash_stack_path = True
+            stack_path = PureWindowsPath(stack_path).as_posix()
```

Comment:
Please create a function that handles this and call it on line 133 like this: `source_url, stack_path = get_paths(data["source_url"], data["stack_path"])`.